src/Sql/Sql/DistributedAvailabilityGroup/Cmdlet/RemoveAzureSqlManagedInstanceLink.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.ManagedInstance.Model;
+using Microsoft.Azure.Commands.Sql.ManagedInstanceHybridLink.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedInstanceHybridLink.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to remove a Managed Instance Link
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceLink",
+        DefaultParameterSetName = DeleteByNameParameterSet,
+        SupportsShouldProcess = true
+        ),
+        OutputType(typeof(AzureSqlManagedInstanceLinkModel))]
```

Comment:
This cmdlet doesn't output any object. Could you check it? BTW, please add -PassThru to allow use to determine whether need output, here is an example: https://github.com/Azure/azure-powershell/blob/fc8e75756f271be1805fc1f61367b8ff1fb80e8d/src/KeyVault/KeyVault/Commands/RemoveAzureKeyVault.cs#L148

Reply:
This one does output the object (confirmed in tests). The reason for that is that I saw many (most) of the Remove-* cmdlets from Sql module don't implement the passthru param, but output the object by default.
I can add the PassThru logic, but I wasn't sure what's the proper way from the UX point of view

Reply:
Updated it anyways

Reply:
Here is an example
![image](https://user-images.githubusercontent.com/19869716/158501955-d6960d84-1bd3-4596-a859-ee2fa2c4f34d.png)

Reply:
Yeah thanks, it's updated to output the object when -PassThru flag is present, otherwise output nothing